docs/detections/alerts-view-details.asciidoc

```diff
@@ -26,7 +26,11 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Reason statement*: Provides a description of what generated the alert and provides general alert details. You can use this to understand the alert's origin and determine if the alert is relevant to your investigation.
+* *Alert-rendered reason statement*: Summarizes alert details in a human-readable format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```

Comment:
@joepeeples this is a fair point. What might be "readable" to one group could be perceived as "non-readable" by others. Switching to "contextual format" could work, but I still feel as though that makes some assumptions, like the user already has the context necessary to understand how they should read the statement -- and some might, but that could be me treating our users as a monolithic group, which they aren't.

@andrew-goldstein @joepeeples do you think "logically-arranged" would work? Maybe something like:

```suggestion
* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```

Reply:
> Switching to "contextual format" could work, but I still feel as though that makes some assumptions, like the user already has the context necessary to understand how they should read the statement 

I meant more that the format establishes the context between the fields, but you're right, this could be easily misconstrued